app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import shutil
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    source_file = parts[1]
+    destination_file = parts[2]
```

Comment:
The code could be optimized.
```suggestion
    source_file, destination_file  = parts[1], parts[2]
```